js/stringProcessing/addWordboundary.js

```diff
@@ -4,16 +4,20 @@
  * Returns a string that can be used in a regex to match a matchString with word boundaries.
  *
  * @param {string} matchString The string to generate a regex string for.
+ * @param {boolean} [positiveLookAhead] Boolean indicating whether or not to include a positive look ahead
+ * for the word boundaries at the end.
  * @param {string} [extraWordBoundary] Extra characters to match a word boundary on.
  * @returns {string} A regex string that matches the matchString with word boundaries.
  */
-module.exports = function( matchString, extraWordBoundary ) {
+module.exports = function( matchString, positiveLookAhead, extraWordBoundary ) {
```

Comment:
Consider using default parameters to be more explicit.

`function( matchString, positiveLookAhead = false, extraWordBoundary = "" )`

Then you can also remove line 14.